describe.js

```diff
@@ -1,4 +1,4 @@
-(function(scope) {
+(function describeFactory(scope, returnInstance) {
```

Comment:
you mean, changing `returnInstance` to `testContext` ? Because it's way more generic than that I think.